examples/Lorenz/Lorenz_example.jl

```diff
@@ -0,0 +1,381 @@
+# Import modules
+include("GModel.jl") # Contains Lorenz 96 source code
+
+# Import modules
+using Distributions  # probability distributions and associated functions
+#using StatsBase
+using LinearAlgebra
+#using StatsPlots
+using GaussianProcesses
+using Plots
+using Random
+using JLD2
+
+# Import Calibrate-Emulate-Sample modules
+using CalibrateEmulateSample.EnsembleKalmanProcesses
+using CalibrateEmulateSample.GaussianProcessEmulator
+using CalibrateEmulateSample.MarkovChainMonteCarlo
+using CalibrateEmulateSample.Observations
+using CalibrateEmulateSample.Utilities
+using CalibrateEmulateSample.ParameterDistributionStorage
+
```

Comment:
Perhaps could you write a short doc-string to explain briefly what is done in the script.
- what the model is
- what parameters you learn 
- what data is used 
- what is the output of the file

Reply:
It should be a bit better now but I'm open to future modifications.

Reply:
It should be a bit better now but I'm open to future modifications.

Reply:
Yes, I will do this.